scripts/e2e-installs.test.js

```diff
@@ -57,14 +56,32 @@ describe('Installation', () => {
             expect(fs.lstatSync(`${appPath}/node_modules`).isDirectory()).toBe(
               true
             );
+
+            expect(() => {
+              execSync(`cd ${appPath} && yarn build`, { stdio: 'ignore' });
+            }).not.toThrow();
           });
         });
       });
     });
 
     describe('Mobile', () => {
+      describe('React InstantSearch Native', () => {
+        test('installs correctly', () => {
+          execSync(
+            `yarn start ${appPath} \
+                --template "React InstantSearch Native"`,
+            { stdio: 'ignore' }
+          );
+
+          expect(fs.lstatSync(`${appPath}/node_modules`).isDirectory()).toBe(
```

Comment:
why is this only done for react native? 🤔

Reply:
It's also done for web templates ([see tests](https://github.com/algolia/create-instantsearch-app/blob/6b15c9471b10ead4a8c867fe61c769711d5b92d1/scripts/e2e-installs.test.js#L50-L58)) but the React Native template doesn't have a `build` script, which is the difference between these two groups of tests.

Does that make sense?